src/main.js

```diff
@@ -0,0 +1,11 @@
+import { TodoListApp } from './utils/todo/todo';
+
+const todoListApp = new TodoListApp({
+  id: '1',
```

Comment:
[Nit] ~~실수로 아이디가 중복될 가능성도 있을 것 같아요.~~
~~유니크한 값을 생성해주는 generator 같은걸 활용하면 좋을 것 같습니다.~~

아이디를 스토리지 키 값으로 사용하고 있어서 바뀌는 상황이 더 곤란하겠네요..! 🤔 

```suggestion
  id: generateId(),
```